File: android/app/src/main/java/co/searchrestaurant/android/app/fetch/GeocodingResponse.java
package co.searchrestaurant.android.app.fetch;

import java.util.List;

/**
 * Created by hassanabid on 1/17/16.
 */
public class GeocodingResponse {

    public String status;
    public List<Result> results;

    public class Result {
        public Geometry geometry;
        public String formatted_address;

    }

    public class Geometry {
        public Location location;
    }

    public class Location {
        public double lat;
        public double lng;
    }


}

File: v2/android/app/src/main/java/dev/hassanabid/searchrestaurant/service/response/GeocodingResponse.kt
package dev.hassanabid.searchrestaurant.service.response

class GeocodingResponse {

    var status: String = ""
    var results: List<Result> = emptyList()

    inner class Result {
        var geometry: Geometry? = null
        var formatted_address: String = ""

    }

    inner class Geometry {
        var location: Location? = null
    }

    inner class Location {
        var lat: Double = 0.toDouble()
        var lng: Double = 0.toDouble()
    }


}